packages/core/docs/scripts/changelog.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const commandArgs = process.argv.slice(2);
+
+const releaseVersion = commandArgs[commandArgs.findIndex(arg => arg === '--v') + 1];
```

Comment:
you can externalize  `commandArgs[commandArgs.findIndex(arg => arg === '...')` into `getCliArgument` as its used in multiple places